docs/src/recipes/aggregate_index.ipynb

```diff
@@ -0,0 +1,151 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c74f7111-1b6c-4454-9770-3f67eeadaca6",
+   "metadata": {},
+   "source": [
+    "# Aggregate events from different indexes\n",
+    "\n",
+    "This recipe applies when you have events indexed by one or more features, and you want to drop some of the index levels and unify the events with the same timestamps.\n",
```

Comment:
What about?

This recipe aggregates possibly non-uniformly sampled events into fixed-length intervals (e.g., seconds, hours, days, or weeks). In other words, it converts the event features into time series.